doc/release-notes-11413.md

```diff
@@ -0,0 +1,10 @@
+Updated or changed RPC
+----------------------
+
+The `bumpfee`, `fundrawtransaction`, `sendmany`, `sendtoaddress`, and `walletcreatefundedpsbt`
+RPC commands have been updated to include two new fee estimation methods "BTC/KB" and "SAT/B".
+The target is the fee expressed explicitly in the given form.
+
+In addition, the estimate_mode parameter is now case insensitive for all of the above RPC commands.
+
+The `bumpfee` command now uses `conf_target` rather than `confTarget` in the options.
```

Comment:
Should mention that `confTarget` is still supported but clients should transition to `conf_target`.

Reply:
I don't really see the point. Literally nobody will be affected by this (i.e. they will either see the release notes and switch, or not see them and not notice the change).